setup.py

```diff
@@ -1,27 +1,54 @@
-from os.path import join, dirname
+import os.path
 
-with open(join(dirname(__file__), 'scrapyd_client/VERSION')) as f:
+from setuptools import find_packages, setup
+
+
+with open('README.rst') as f:
+    readme = f.read()
```

Comment:
For Windows support, I recommend explicitly using UTF-8 encoding here.

Reply:
Good catch! Done in 2a14f81